features/providers/step_definitions/civil_journey_steps.rb

```diff
@@ -298,6 +330,20 @@
   expect(page).to have_content('Other assets')
 end
 
+Then('I am on the read only version of the check your answers page') do
+  expect(page).to have_content('Home')
+  expect(page).not_to have_content('change')
+  expect(page).not_to have_content('Continue')
+  expect(page).not_to have_content('Back')
+  expect(page).not_to have_content('Check your answers')
```

Comment:
This feels link you are testing how the page is behaving rather than you checking that this is the right page. By adding so many checks, this makes the condition much more fragile (a content change to any of those text entries could cause a failure).

If you need to test that when the page is in Read-Only mode, that the page displays the context as you have set out here, then do that in a request spec and not in the cucumber feature.

Reply:
Changed to the below because the read only page currently has no page title. therefore, I needed to find something within the page to prove that i've landed on the correct page.

Added:
  expect(page).to have_content('Home')
  expect(page).not_to have_css('.change-link')